dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -76,6 +84,9 @@ private static void buildDefaultData() throws DotDataException {
 		} catch (DotSecurityException e) {
 			Logger.fatal(DotCMSInitDb.class, "Unable to initialize default data", e);
 			throw new DotRuntimeException(e.getMessage(), e);
+		} catch (Exception e) {
+			Logger.fatal(DotCMSInitDb.class, "Unable to initialize default data", e);
+			new DotDataException(e.getMessage(), e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Throw this exception or remove this useless statement [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3984)